middleware/logger/config.go

```diff
@@ -115,5 +120,11 @@ func configDefault(config ...Config) Config {
 	if cfg.Output == nil {
 		cfg.Output = ConfigDefault.Output
 	}
+	// Set custom tags
+	if len(cfg.CustomTags) != 0 {
+		for k, v := range cfg.CustomTags {
+			tagMap[k] = v
```

Comment:
think this location is not safe when used with multiple logger instances for different routes

if you initialize multiple loggers with different configurations for different routes or apps to be mounted, then we would find our customTags functions from other functions here again

i.e. a custom tag in another function overwrites the normal tag function globally

we should always set only the customTag map and use it and rename it to "tagFunctions" if necessary